Rename copy keybinding arg (#5216)
microsoft/terminal: doc/cascadia/SettingsSchema.md

| `closePane` | Close the active pane. | | | |
| `closeTab` | Close the current tab. | | | |
| `closeWindow` | Close the current window and all tabs within it. | | | |

> | `copy` | Copy the selected terminal content to your Windows Clipboard. | `singleLine` | boolean | When `true`, the copied content will paste on one line. When `false`, newlines persist from the selected text. |

Reviewer: ```suggestion
| `copy` | Copy the selected terminal content to your Windows Clipboard. | `singleLine` | boolean | When `true`, the copied content will be copied as a single line. When `false`, newlines persist from the selected text. |
```
